Module_0_Fundamentals/02_Python_Libraries_for_Data_Science/01_Numpy/02_Arrays.ipynb

```diff
@@ -48,13 +44,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "vscode": {
-     "languageId": "plaintext"
+   "execution_count": 1,
+   "metadata": {},
```

Comment:
@jaimelaraglobant will this change introduce any error on the numpy notebook?

Reply:
@mike-peraza on my local is working well, and the changes as I see are auto-placed when the code ran n times is only a counter